gradle/libs.versions.toml

```diff
@@ -0,0 +1,68 @@
+[versions]
+bytebuddy = "1.12.10"
+compose-compiler = "1.3.0"
+java-target = "1.8"
+jcodec = "0.2.5"
+kotlin = "1.7.10"
+ktlint = "0.46.1"
+moshi = "1.13.0"
+
+# Maps to this commit: https://android.googlesource.com/platform/prebuilts/studio/layoutlib/+/fa3aa65
+# which runs on JDK 11.
+layoutlib = "2021.2.1-patch1-fa3aa65"
+layoutlibPrebuiltSha = 'fa3aa65'
```

Comment:
nit: double quotes?

Reply:
huh, didn't know it works both ways, fixed